Sources_v3/Database/DTOs/CurrentUserDTO.swift

```diff
@@ -58,12 +58,24 @@ extension NSManagedObjectContext: CurrentUserDatabaseSession {
         dto.mutedUsers = [] // TODO: mutedUsers
         dto.user = try saveUser(payload: payload)
         
-        // TODO: unread counts
+        if let unreadCount = payload.unreadCount {
+            try saveCurrentUserUnreadCount(count: unreadCount)
+        }
+        
         // TODO: devices
         
         return dto
     }
     
+    func saveCurrentUserUnreadCount(count: UnreadCount) throws {
+        guard let dto = currentUser() else {
+            throw ClientError("Current user doesn't exist")
```

Comment:
There's already `ClientError.CurrentUserDoesNotExist()`

Reply:
fixed in a5002aa28109294765a23d548da3befb76427722